packages/loaders/src/styled/StyledInline.ts

```diff
@@ -5,18 +5,22 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled, { keyframes } from 'styled-components';
+import styled, { DefaultTheme, ThemeProps, keyframes } from 'styled-components';
 import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'loaders.inline';
 
-const PULSE_ANIMATION = keyframes`
-  0%, 100% {
-    opacity: .2;
+const retrieveAnimation = ({ theme }: ThemeProps<DefaultTheme>) => keyframes`
+  0% {
+    opacity: 1;
   }
 
   50% {
-    opacity: .8;
+    opacity: ${theme.opacity[600]};
+  }
+
+  100% {
+    opacity: ${theme.opacity[200]};
   }
```

Comment:
Why did the 0% portion of the animation change?

Reply:
Aligning with Figma, where the first dot has 100% opacity.

![Screenshot 2024-06-06 at 7 26 45 AM](https://github.com/zendeskgarden/react-components/assets/6879688/0be9ad0f-f347-469a-b7d6-6d5d389b322c)

Reply:
Ok, I checked with @lucijanblagonic and he wasn't expecting the animation to materially change based on Figma. And now we're seeing some odd animation with the `Inline` loader. I think we should revert here (while retaining the `theme.opacity` usage).